packages/sdk/src/constants/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Amount lower than this is considered as dust value
+ * and majority of the miners don't pick txs w/ the following output value or lower
+ */
+export const MINIMUM_AMOUNT_IN_SATS = 600;
+
+/**
+ * Fee calculated by the fee estimator cannot be greater than 0.05 BTC in any case
+ */
+export const MAXIMUM_FEE = 5000000;
```

Comment:
for bigger numbers maybe we split with `_` for better readability. so `5_000_000`